js/src/widgets/editorPanel.js

```diff
@@ -68,6 +68,8 @@
 
               return false;
             });
+            var annos = { data: state.annotations };
+            jQuery.publish('annotationsListFiltered' + this.windowId, annos);
```

Comment:
I struggled with which module should be publishing this too.  The reason I stuck it there was because that's where the list filtering logic already lived.  I was thinking that the annotationsLayer might provide an easier way to jump among lists as well (and that manages the non-filtered list data), but that would have required a decent amount of rewriting.